docs/conf.py

```diff
@@ -21,6 +21,12 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath('..'))
 
+# configuration required to import test modules
+for path in ["../pyanaconda/isys/.libs", "../pyanaconda", "../tests", "../tests/lib", "../dracut", "../widgets"]:
```

Comment:
I'm not sure if these "raw" (non-prefixed)  relative paths are a good way to go. Or do they work the same no matter which directory you run the sphinx stuff from?